src/Data/List/Membership/DecSetoid.agda

```diff
@@ -9,10 +9,11 @@
 open import Relation.Binary.Definitions using (Decidable)
 open import Relation.Binary.Bundles using (DecSetoid)
 open import Relation.Nullary.Decidable using (¬?)
+open import Relation.Nullary using (Stable; decidable-stable)
+open import Data.List.Relation.Unary.Any using (any?)
```

Comment:
Of these, L10 is the only one required for the opening module header L15 to typecheck, so `style-guide` [requires](https://github.com/agda/agda-stdlib/blob/59d7b2ec356d062245a0fdcd8141b3fca404023d/doc/style-guide.md?plain=1#L105-L121) moving all the other `import`s *below* that line...

Reply:
I would prefer the inverse policy:
> Only put imports below the parametrized module that actually get instantiated with the module params.

But since you've already decided, I'll go with the flow.

The alphabetical requirement must be a joke, right?
> * The list of imports should be declared in alphabetical order.